pre-commit.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+from typing import List
+import subprocess
+import sys
+import enum
+import argparse
+import os
+
+
+@enum.unique
+class Color(enum.Enum):
+    RED = "\033[0;31m"
+    GREEN = "\033[0;33m"
+    CYAN = "\033[0;36m"
+
+
+NC = "\033[0m"  # No Color
```

Comment:
nitpick: I like this change! However, I think `NC` is somewhat isolated from the other color options; maybe it could be `Color.None` or `Color.Blank`. Or you could move it to `colorify`. Or leave it be :)

Reply:
I've decided to separate it because it is an invalid choice for color argument of `colorify` but IMO it is convenient to see it next to proper colors, so just next to colors enum